src/extension/android/androidPlatform.ts

```diff
@@ -66,13 +68,32 @@ export class AndroidPlatform extends GeneralMobilePlatform {
     constructor(protected runOptions: IAndroidRunOptions, platformDeps: MobilePlatformDeps = {}) {
         super(runOptions, platformDeps);
         this.adbHelper = new AdbHelper(this.runOptions.projectRoot, this.logger);
+        this.emulatorManager = new AndroidEmulatorManager(this.adbHelper);
     }
 
     // TODO: remove this method when sinon will be updated to upper version. Now it is used for tests only.
     public setAdbHelper(adbHelper: AdbHelper) {
         this.adbHelper = adbHelper;
     }
 
+    public async startEmulatorIfNotRun(target: string): Promise<any> {
```

Comment:
```suggestion
    public async startEmulator(target: string): Promise<any> {
```

Reply:
Also, I think it is better to explicitly return `AndroidEmulator` interface instance `{name, id}` or null instead of any.